### PR TITLE
Enhanced parser for local routing table

### PR DESCRIPTION
**Pull request by vishwanathjadhav**
Enhanced parser for the local routing table. If the machine has  IP addresses on Ethernet interface, there will be a route to each locally hosted IP in the local routing table. The local routing table is important for one of the insights-rule, hence enhancing the parser.

### CODE REVIEW

**Comment by xiangce** [comment]
@vishwanathjadhav - would you please to open an issue for this enhancement at first? normally for any enhancement or bugfix, a related issue that explains the detailed reason is necessary.

File: insights/parsers/ip.py [hunk 1]
```diff
@@ -325,7 +325,6 @@ class RouteDevices(CommandParser):
     # Why have types we ignore?
     IGNORE_TYPES = set(["broadcast",
                         "throw",
-                        "local",
```
Comment (xiangce) [inline insights/parsers/ip.py]: I think better move the "local" to the `SAVED_TYPES`

File: insights/parsers/ip.py [hunk 2]
```diff
@@ -405,6 +404,8 @@ def parse_route(self, parts):
         table_type = None
         if parts[0] in self.IGNORE_TYPES:
             return None
+        if parts[0] == 'local':
```
Comment (xiangce) [inline insights/parsers/ip.py]: By moving the `local` to `SAVED_TYPES`, this logic won't be changed.

File: insights/parsers/ip.py [hunk 3]
```diff
@@ -280,7 +280,7 @@ class RouteDevices(CommandParser):
     * ``netmask`` - the CIDR notation of the network as a string (e.g. '24')
     * ``table`` - the routing table, or None if not given.
 
-    Destinations given as **broadcast**, **throw**, **local**,
+    Destinations given as **broadcast**, **throw**,
     **unreachable**, **prohibit**, **blackhole** and **nat** are discarded.
 
     Destinations given as **unicast**, **multicast**
```
Comment (bfahr) [inline insights/parsers/ip.py]: @vishwanathjadhav this docstring line is incomplete, looks like its missing the rest of the sentence.

File: insights/parsers/ip.py (hunk 1 above)
Comment (vishwanathjadhav) [reply insights/parsers/ip.py]: Done

File: insights/parsers/ip.py (hunk 2 above)
Comment (vishwanathjadhav) [reply insights/parsers/ip.py]: Done

File: insights/parsers/ip.py (hunk 3 above)
Comment (vishwanathjadhav) [reply insights/parsers/ip.py]: @bfahr - no, the line is complete, just required to adjust the line. The changes are done.

Comment (xiangce) [reply insights/parsers/ip.py]: @vishwanathjadhav - I think what @bfahr pointed out is the line-286, but not the line-283, would you please to help complete the line-286?

Comment (vishwanathjadhav) [reply insights/parsers/ip.py]: @xiangce , @bfahr - Thanks, I have update the doc

**Comment by xiangce** [comment]
@vishwanathjadhav - please check this line, if it's OK for `local` is in SAVED_TYPES

**Comment by xiangce** [comment]
@vishwanathjadhav - I opened an issue for this check. we can skip it in this PR.

**Pull request merged**